wiki/Manual.md

```diff
@@ -3733,15 +3733,29 @@ if ($exchange->has['createMarketOrder']) {
 
 #### Market Buys
 
-In general, when placing a `market buy` or `market sell` order the user has to specify just the amount of the base currency to buy or sell. However, with some exchanges market buy orders implement a different approach to calculating the value of the order.
+When placing a **market buy** or **market sell** order the user usually has to specify just the amount of the **quote** currency to buy or sell.
```

Comment:
hum, I would say it's the other way around

Reply:
in an `ADA/USDT` order, you specify an amount of 4 to buy 4 ADA